WordPress/src/test/java/org/wordpress/android/viewmodel/reader/ReaderPostListViewModelTest.kt

```diff
@@ -36,12 +49,60 @@ class ReaderPostListViewModelTest {
                 reblogUseCase,
                 readerTracker,
                 readerSeenStatusToggleUseCase,
+                quickStartRepository,
+                selectedSiteRepository,
                 TEST_DISPATCHER,
                 TEST_DISPATCHER
         )
     }
 
+    /* QUICK START */
+
     @Test
-    fun foo() {
+    fun `when quick start follow site task event received, then follow site setting step prompt shown`() {
+        val observers = initObservers()
+
+        viewModel.onQuickStartEventReceived(QuickStartEvent(QuickStartTask.FOLLOW_SITE))
+
+        val quickStartReaderPrompt = observers.quickStartReaderPrompts.last().peekContent()
+        assertThat(quickStartReaderPrompt).isEqualTo(QuickStartReaderPrompt.FollowSiteSettingsStepPrompt)
     }
+
+    @Test
+    fun `when quick start non follow site event received, then follow site setting step prompt not shown`() {
```

Comment:
Nitpick: Does `when quick start event received is not follow site` better than `when quick start non follow site event received`?. 🤔

Reply:
Updated all test cases to follow this convention: f97545d